lib/ansible/modules/database/postgresql/postgresql_query.py

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: postgresql_query
+short_description: Run arbitrary queries on a postgresql database.
+description:
+   - Run arbitrary SQL queries against a PostgreSQL instance from the current host.
+   - Query results are returned as "query_results" as a list of dictionaries.
+     Number of rows affected are returned as "row_count".
+   - Can read queries from a .sql script files.
+   - SQL scripts may be templated with ansible variables at execution time.
```

Comment:
Should have a note here that check mode will run the query and then roll it back. Something like:
```
- Ansible check mode will run the query and then roll it back. 
  Be careful using long running queries if you're going to use check mode.
```